docs/usage.md

```diff
@@ -1,103 +1,480 @@
+    | InvalidManagedPolicy01 | AWS::IAM::ManagedPolicy | infra_cfn/invalid_long_description.yaml | Low      | CUSTOM_0001 | long_description_cfn | IAM policies must have a description of at least 25 characters | FAIL   |
+    | ValidManagedPolicy01   | AWS::IAM::ManagedPolicy | infra_cfn/invalid_long_description.yaml | Low      | CUSTOM_0001 | long_description_cfn | IAM policies must have a description of at least 25 characters | PASS   |
+    +------------------------+-------------------------+-----------------------------------------+----------+-------------+----------------------+----------------------------------------------------------------+--------+
+    |                        |                         |                                         |          |             |                      |                                                        Overall |   FAIL |
+    +------------------------+-------------------------+-----------------------------------------+----------+-------------+----------------------+----------------------------------------------------------------+--------+
+    ```
+
+=== "junit"
+
+    ```
+    <testsuites name="Regula">
+      <testsuite name="infra_cfn/invalid_long_description.yaml" tests="2">
+        <testcase name="InvalidManagedPolicy01" classname="AWS::IAM::ManagedPolicy" assertions="1">
+          <failure message="IAM policies must have a description of at least 25 characters" type="long_description_cfn">Rule ID: CUSTOM_0001&#xA;Rule Name: long_description_cfn&#xA;Severity: Low&#xA;Message: IAM policies must have a description of at least 25 characters</failure>
+        </testcase>
+        <testcase name="ValidManagedPolicy01" classname="AWS::IAM::ManagedPolicy" assertions="1"></testcase>
+      </testsuite>
+    </testsuites>
+    ```
+
+=== "tap"
+
+    ```
+    not ok 0 InvalidManagedPolicy01: IAM policies must have a description of at least 25 characters
+    ok 1 ValidManagedPolicy01: IAM policies must have a description of at least 25 characters
+    ```
+
+For more about Regula's output, see [Report Output](report.md).
+
+## repl
+
+```
+Start an interactive session for testing rules with Regula
+
+Usage:
+  regula repl [rego paths] [flags]
+
+Flags:
+  -h, --help        help for repl
+  -u, --user-only   Disable built-in rules
+```
+
+`regula repl` is the same as OPA's REPL ([`opa run`](https://www.openpolicyagent.org/docs/latest/#3-try-opa-run-interactive)), but with the Regula library and ruleset built in (unless you disable it with `--user-only`). Additionally, Regula's REPL allows you to generate [`mock_input`, `mock_resources`, and `mock_config`](development/rule-development.md#test-inputs) at runtime.
+
+To view this dynamically generated data, start by loading one or more IaC files or a directory containing IaC files. Note that Regula will only operate on individual files rather than interpreting the entire directory as a single configuration.
```

Comment:
> Note that Regula will only operate on individual files rather than interpreting the entire directory as a single configuration.

I don't know if it's necessary to clarify it here, but this sentence only applies to the `repl` and `test` commands. We do it because the main use-case for this feature is to load test configurations during rule development.

I think we could definitely leave this as-is, but wanted to highlight it in case we want clarification here.

Reply:
I agree -- an important clarification!